discord/guild.py

```diff
@@ -1235,6 +1235,25 @@ async def edit(self, *, reason=None, **fields):
                 fields['public_updates_channel_id'] = public_updates_channel
             else:
                 fields['public_updates_channel_id'] = public_updates_channel.id
+
+        features = []
+
+        try:
+            community = fields['community']
+        except KeyError:
+            community = 'COMMUNITY' in self.features
+
+        if community:
+            try:
+                fields['rules_channel_id']
+                fields['public_updates_channel_id']
+            except KeyError:
+                raise InvalidArgument('community field requires both rules_channel and public_updates_channel fields to be provided') from None
+            else:
+                features.append('COMMUNITY')
+
+        fields['features'] = features
```

Comment:
I think this implementation removes the community feature if you don't pass it at all.